patch_rebaser/patch_rebaser.py

```diff
@@ -81,8 +81,13 @@ def get_patches_branch(repo, remote, dlrn_projects_ini):
     # Get downstream distgit branch from DLRN config
     distgit_branch = get_downstream_distgit_branch(dlrn_projects_ini)
 
-    # Guess at patches branch based on the distgit branch name
-    return find_patches_branch(repo, remote, distgit_branch)
+    branch_name = os.environ.get('PATCHES_BRANCH', None)
+    if branch_name:
+        return branch_name
```

Comment:
I think we need to check the branch exists before returning it, like on line 32, and return None if not. Otherwise, we will miss the chance to create it on line 455.

Reply:
Thanks for that! Right, it did not check if the branch existed. I've fixed that in the following version.